app/grants/views.py

```diff
@@ -3397,9 +3398,28 @@ def ingest_contributions(request):
     profile = request.user.profile
     txHash = request.POST.get('txHash')
     userAddress = request.POST.get('userAddress')
+    signature = request.POST.get('signature')
+    message = request.POST.get('message')
     network = request.POST.get('network')
     ingestion_types = [] # after each series of ingestion, we append the ingestion_method to this array
 
+    # Setup web3
+    PROVIDER = f"wss://{network}.infura.io/ws/v3/{settings.INFURA_V3_PROJECT_ID}"
+    w3 = Web3(Web3.WebsocketProvider(PROVIDER))
```

Comment:
`w3 = get_web3(network)`

Reply:
Just pushed 9847312 to update this if you want to retest > merge > deploy